helm/wind-river-cloud-platform-deployment-manager/templates/manifests.yaml

```diff
@@ -599,6 +607,20 @@ spec:
   secretName: {{ include "helm.name" . }}-webhook-server-secret
 ---
 apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ .Values.namespace }}-metrics-cert
+  namespace: {{ .Values.namespace }}
+spec:
+  dnsNames:
+  - {{ .Values.namespace }}-metrics-service.{{ .Values.namespace }}.svc
```

Comment:
Just a question: can we update the existing serving-cert with the metrics service into the dnsNames? I would expect it can simplify this process, and we can avoid creating a new additional cert